src/sass/partials/layouts/_content-card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 2rem 0 0;
+  width: 100%;
+
+
+  @include breakpoint(setting-get('two grid cards')) {
+    margin-right: 4%;
+    width: 48%;
```

Comment:
Updated to `padding` instead of `margin` and mirrored the `rhythm` to what we are already using on the `form--field-container`. With that said, 48% is used since we have a border and need there to be gutter between the cards.